src/pybind/mgr/dashboard/tox.ini

```diff
@@ -58,13 +70,30 @@ dirs =
     HACKING.rst
 
 [testenv:lint]
-basepython = python3
+basepython=python3
 deps =
     -rrequirements-lint.txt
 commands =
-    rstcheck --report info --debug {posargs:{[rstlint]dirs}}
-    flake8 {posargs}
-    pylint {[pylint]addopts} {posargs:{[pylint]dirs}}
+    lint: flake8 {posargs}
+    lint: pylint {[pylint]addopts} {posargs:{[pylint]dirs}}
+    lint: rstcheck --report info --debug {posargs:{[rstlint]dirs}}
+
+
+[autopep8]
+addopts =
+    --max-line-length {[flake8]max-line-length}
+    --in-place
+    --recursive
```

Comment:
For this to work in my environment I need to add this option:
```--exclude "{[flake8]exclude}"```
otherwise it crashes.
My environment: docker CentOS 7
How to reproduce it:
```
cd src/pybind/mgr/dashboard
# tox -e fix
fix installed: astroid==2.2.5,autopep8==1.4.4,docutils==0.15.2,entrypoints==0.3,flake8==3.7.8,flake8-colors==0.1.6,isort==4.3.21,lazy-object-proxy==1.4.2,mccabe==0.6.1,pycodestyle==2.5.0,pyflakes==2.1.1,pylint==2.3.1,rstcheck==3.3.1,six==1.12.0,typed-ast==1.4.0,wrapt==1.11.2
fix run-test-pre: PYTHONHASHSEED='2684153044'
fix run-test: commands[0] | autopep8 --max-line-length 100 --in-place --recursive .
Traceback (most recent call last):
  File "/ceph/src/pybind/mgr/dashboard/.tox/fix/bin/autopep8", line 10, in <module>
    sys.exit(main())
  File "/ceph/src/pybind/mgr/dashboard/.tox/fix/lib/python3.6/site-packages/autopep8.py", line 4200, in main
    ret = fix_multiple_files(args.files, args, sys.stdout)
  File "/ceph/src/pybind/mgr/dashboard/.tox/fix/lib/python3.6/site-packages/autopep8.py", line 4100, in fix_multiple_files
    ret = _fix_file((name, options, output))
  File "/ceph/src/pybind/mgr/dashboard/.tox/fix/lib/python3.6/site-packages/autopep8.py", line 4080, in _fix_file
    return fix_file(*parameters)
  File "/ceph/src/pybind/mgr/dashboard/.tox/fix/lib/python3.6/site-packages/autopep8.py", line 3454, in fix_file
    fixed_source = fix_lines(fixed_source, options, filename=filename)
  File "/ceph/src/pybind/mgr/dashboard/.tox/fix/lib/python3.6/site-packages/autopep8.py", line 3434, in fix_lines
    fixed_source = fix.fix()
  File "/ceph/src/pybind/mgr/dashboard/.tox/fix/lib/python3.6/site-packages/autopep8.py", line 593, in fix
    aggressive=self.options.aggressive))
  File "/ceph/src/pybind/mgr/dashboard/.tox/fix/lib/python3.6/site-packages/autopep8.py", line 535, in _fix_source
    modified_lines = fix(result)
  File "/ceph/src/pybind/mgr/dashboard/.tox/fix/lib/python3.6/site-packages/autopep8.py", line 889, in fix_e402
    line_index)
  File "/ceph/src/pybind/mgr/dashboard/.tox/fix/lib/python3.6/site-packages/autopep8.py", line 1442, in get_module_imports_on_top_of_file
    if cnt == import_line_index or is_future_import(line):
  File "/ceph/src/pybind/mgr/dashboard/.tox/fix/lib/python3.6/site-packages/autopep8.py", line 1412, in is_future_import
    nodes = ast.parse(line)
  File "/usr/lib64/python3.6/ast.py", line 35, in parse
    return compile(source, filename, mode, PyCF_ONLY_AST)
  File "<unknown>", line 2
    'local',
    ^
IndentationError: unexpected indent
ERROR: InvocationError for command /ceph/src/pybind/mgr/dashboard/.tox/fix/bin/autopep8 --max-line-length 100 --in-place --recursive . (exited with code 1)
_______________________________________________________________________________________________________________________ summary _______________________________________________________________________________________________________________________
ERROR:   fix: commands failed
```

Reply:
@alfonsomthd just tried in a CentOS7 environment and it worked:
```bash
# lsb_release -a
LSB Version:	:core-4.1-amd64:core-4.1-noarch
Distributor ID:	CentOS
Description:	CentOS Linux release 7.6.1810 (Core) 
Release:	7.6.1810
Codename:	Core

# tox -e fix
fix create: /ceph/src/pybind/mgr/dashboard/.tox/fix
fix installdeps: -rrequirements-lint.txt
fix installed: tox -e fix
astroid==2.2.5,autopep8==1.4.4,docutils==0.15.2,entrypoints==0.3,flake8==3.7.8,flake8-colors==0.1.6,isort==4.3.21,lazy-object-proxy==1.4.2,mccabe==0.6.1,pycodestyle==2.5.0,pyflakes==2.1.1,pylint==2.3.1,rstcheck==3.3.1,six==1.12.0,typed-ast==1.4.0,wrapt==1.11.2,You are using pip version 9.0.1, however version 19.2.2 is available.,You should consider upgrading via the 'pip install --upgrade pip' command.
fix runtests: PYTHONHASHSEED='405480326'
fix runtests: commands[0] | python --version
Python 3.6.8
fix runtests: commands[1] | autopep8 --max-line-length 100 --in-place --recursive .
__________________________________________________________________ summary __________________________________________________________________
  fix: commands succeeded
  congratulations :)
``` 

Nevertheless, as based on recent issues posted to `autopep8` Github repo it looks that there might be some issue there, I'll proceed with your suggestion.